src/MapfileStyleParser.ts

```diff
@@ -824,6 +831,24 @@ export class MapfileStyleParser implements StyleParser {
     return symbolizers;
   }
 
+  /**
+   * Collect icon data if defined within the label tag
+   *
+   * @param {MapfileLabel} mapfileLabel Mapfile label data
+   * @return {IconSymbolizer[]} The IconSymbolizer
+   */
+  getIconsFromMapfileLabel(mapfileLabel: MapfileLabel): IconSymbolizer[] {
+    const labelStyles: IconSymbolizer[] = [];
```

Comment:
very minor and definitely non blocking: shouldn't this rather be `iconStyles`?

Reply:
Renamed the variable.